packages/react-devtools-shared/src/config/DevToolsFeatureFlags.extension-oss.js

```diff
@@ -16,6 +16,8 @@
 export const enableProfilerChangedHookIndices = true;
 export const isInternalFacebookBuild = false;
 export const enableNamedHooksFeature = true;
+export const enableLogger = false;
+
```

Comment:
nit: Any reason for the extra empty line here?

Reply:
oops, sorry i think this was due to resolving merge conflicts